projects/workshop/desafio_rick_and_morty/README.md

```
@@ -0,0 +1,210 @@
+
+## ▶️ Rodando o projeto localmente
+
+### 1) Clonar e instalar dependências
+```bash
+git clone <URL_DO_REPO>
+navegue até a o diretorio do projeto ../projects/workshop/desafio_rick_and_morty
+cd rnm_app
+flutter pub get
+```
+
+### 2) Executar (Android/iOS/Web)
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)